src/app/View.ts

```diff
@@ -0,0 +1,123 @@
+import { Render } from './utils/Render';
+import { Timer } from './Timer';
+import { AnimalNames } from '../Enums/AnimalNamesEnum';
+import { Player } from '../Player';
+
+export class View {
+  renderMenuView(): void {
+    const inputName = Render.elementFactory('input', {
+      className: 'input__name',
+      placeholder: 'Carlos Santana',
+    });
+    const startGameButton = Render.elementFactory(
+      'button',
+      {
+        className: 'button__start button',
+      },
+      'Start Game',
+    );
+    const avatarsElements = Object.values(AnimalNames).map(
+      (animalName) => {
+        if (
+          animalName === AnimalNames.BIG_DOG ||
+          animalName === AnimalNames.SMALL_DOG
+        ) {
+          return Render.elementFactory('img', {
+            src: `./static/images/avatars/dog.png`,
+          });
+        }
+
+        return Render.elementFactory('img', {
+          src: `./static/images/avatars/${animalName}.png`,
+        });
+      },
+    );
+    avatarsElements.splice(5, 1);
+    Render.render(
+      '#sf-app',
+      inputName,
+      startGameButton,
+      ...avatarsElements,
+    );
+    let playersChosenAvatarPath = '';
+    avatarsElements.forEach((el) => {
+      el.addEventListener('click', (e): void => {
+        playersChosenAvatarPath = (e.target as any).src;
+      });
+    });
+
```

Comment:
`      el.addEventListener('click', (e: Event): void => {
        const avatarClicked = e.target as HTMLImageElement;
        playersChosenAvatarPath = avatarClicked.src;
        avatarClicked.style.border = '4px solid lightgrey';`

Reply:
For now, as it is still a solution for Sprint I, we don't need any styling. The only thing we care about is that it should work.

Reply:
Yes, it doesn't need complicated styling for now, but there will be no confirmation that anything was clicked or chosen, if no styles change on click